docs/ilm.asciidoc

```diff
@@ -0,0 +1,190 @@
+[[ilm]]
+[role="xpack"]
+== Index lifecycle management (ILM)
+
+Use the {ref}/getting-started-index-lifecycle-management.html[index lifecycle management]
+feature in {es} to manage your APM Server indices as they age.
+For example, instead of creating daily indices where index size can vary based on the number of agents and number of events sent,
+use an index lifecycle policy to automate a rollover to a new index when the existing index reaches a specified size or age.
+
+You can view and edit the index lifecycle policies in the *Index lifecycle policies* UI in {kib}.
+For more information about working with the UI,
+see {kibana-ref}/index-lifecycle-policies.html[Index lifecyle policies].
+
+NOTE: If you're migrating from an existing setup,
+any indices present before ILM was configured will need to be managed manually.
+
+[float]
+[[ilm-setup]]
+=== ILM default configuration
+
+IMPORTANT: The default index lifecycle configuration cannot be changed until after it has been applied.
```

Comment:
Once the indices are created, changes to templates and policies are not going to be applied to existing indices. 

With ILM in APM being relatively strict and not offering config options at the moment, I'd generally head into the direction of _In case you'd like to manage your own ILM policies refer to our manual setup guide_.